index.html

```diff
@@ -2,9 +2,20 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title></title>
+    <link rel="stylesheet" href="./app.css">
+    <title>Popular Github</title>
   </head>
   <body>
-
+      <div class="profile-photo">
+        <div class="profile-name">
```

Comment:
Watch out for closing tags correctly. Ask yourself, is the screenname a sub element of the name and is that then a sub element of the photo? Or should these be text style elements or something more semantic next to each other and maybe grouped using divs to make it a bit more outline like.

Remind me tomorrow and I'll show a quick way to help visualize the "outline" and grouping.